go.mod

```diff
@@ -109,4 +109,4 @@ replace k8s.io/client-go v12.0.0+incompatible => k8s.io/client-go v0.21.4
 
 // Replace memberlist with our fork which includes some fixes that haven't been
 // merged upstream yet.
-replace github.com/hashicorp/memberlist => github.com/grafana/memberlist v0.3.1-0.20220708130638-bd88e10a3d91
+replace github.com/hashicorp/memberlist => github.com/grafana/memberlist v0.3.1-0.20220714140823-09ffed8adbbe
```

Comment:
Is this necessary??

Reply:
Not necessary per se, but upgraded anyway because I saw some latest version available for "grafana/memberlist" and only place it's used is dskit/ring AFAIK.

I can remove if we have any specific reason not to upgrade.

Reply:
OK, let me just check with others with more knowledge of the memberlist repo.

Reply:
I've confirmed that this is the master revision of grafana/memberlist, and it's fine to upgrade to :+1: